ballerina/http_client.bal

```diff
@@ -127,40 +125,40 @@ client isolated class HttpClient {
         return externSubmit(self, httpVerb, path, <Request>message);
     }
 
-    # Retrieves the `http:Response` for a previously-submitted request.
+    # Get the response from a previously submitted asynchronous request. Can be used after calling `submit()` action to
+    # retrieve the actual response.
```

Comment:
Can't we have this as a single line? Other Document titles seems to be in a single line

Reply:
+1